kubernetes/adapter.go

```diff
@@ -165,15 +167,21 @@ func newIngressForKube(i *Ingress) *ingress {
 }
 
 // NewAdapter creates an Adapter for Kubernetes using a given configuration.
-func NewAdapter(config *Config) (*Adapter, error) {
+func NewAdapter(config *Config, ingressClassFilters []string) (*Adapter, error) {
 	if config == nil || config.BaseURL == "" {
 		return nil, ErrInvalidConfiguration
 	}
 	c, err := newSimpleClient(config)
 	if err != nil {
 		return nil, err
 	}
-	return &Adapter{kubeClient: c}, nil
+	return &Adapter{kubeClient: c, ingressFilters: ingressClassFilters}, nil
+}
+
+// Get ingress class filters that are used to filter ingresses acted upon.
+func (a *Adapter) IngressFiltersString() string {
+        result := fmt.Sprintf("%s", strings.Join(a.ingressFilters, ","))
```

Comment:
you don't need Sprintf here, just pass strings.Join() into strings.TrimSpace() :)

Reply:
Will adjust, thanks.